### PR TITLE
0.10.0 release notes

### PR DESCRIPTION
**Pull request by sjcorbett**
Phew!

### CODE REVIEW

**Comment by sjcorbett** [comment]
P.S. I put together a really quick script for retrieving the PRs that were merged into brooklyn-server that may be useful the next time someone wants to write release notes: https://gist.github.com/sjcorbett/72ed944b06ce3a138fbe516e8d36f624.

File: guide/misc/release-notes.md [hunk 1]
```diff
@@ -14,15 +14,121 @@ to Brooklyn's commercial users for funding much of this development.
...
+
+Additionally, location customisers can now be attached to entities as
+initialisers, rather than having to dig into their `templateOptions`.
+
+
+#### Brooklyn server
+
+* The REST API requires CSRF headers for non-GET and HEAD requests when
+  sessions are in effect.
+  See [CsrfTokenFilter](https://brooklyn.apache.org/v/0.10.0/misc/javadoc/org/apache/brooklyn/rest/filter/CsrfTokenFilter.html)
```
Comment (sjcorbett) [inline guide/misc/release-notes.md]: I'm assuming that this URL is going to exist when the release is done.

File: guide/misc/release-notes.md [hunk 2]
```diff
@@ -14,15 +14,121 @@ to Brooklyn's commercial users for funding much of this development.
...
+  for details.
+
+* The catalogue scanner can be configured with whitelists and blacklists
+  of bundle IDs to control which bundles may present applications in the catalogue.
+
+* The Karaf distribution includes equivalents of the
+  [cloud explorer commands](https://brooklyn.apache.org/v/0.10.0/ops/server-cli-reference.html#cloud-explorer).
```
Comment (sjcorbett) [inline guide/misc/release-notes.md]: I'm assuming this URL will exist too.

**Comment by neykov** [comment]
Awesome work @sjcorbett. Some minor comments - see what you think, then good to merge.

File: guide/misc/release-notes.md (hunk 2 above)
Comment (sjcorbett) [inline guide/misc/release-notes.md]: I think this should point at 0.10.0 specifically.

**Comment by sjcorbett** [comment]
@neykov thanks for checking. I've removed the reference to `getImmediately`.

File: guide/misc/release-notes.md (hunk 2 above)
Comment (neykov) [inline guide/misc/release-notes.md]: Agree. The example I was referring to is:
```
{{ site.path.v }}/0.9.0/misc/release-notes.html
```
This makes it possible to have the docs live under a different root, possible a local one for development, testing. Or a staging area for the docs.

Comment (sjcorbett) [inline guide/misc/release-notes.md]: Got you.

**Comment by neykov** [comment]
Thanks @sjcorbett. I've merged this in the [0.10.0 branch](https://github.com/apache/brooklyn-docs/commits/0.10.0). Can you close the PR.

**Pull request closed without merging**